docs/requirements/requirements.md

```diff
@@ -299,6 +316,12 @@ This document is in the making since we are still in the requirements engineerin
 - The systems' microservices must not fail suddenly and without good reason.
 - Origin: Researcher *Toni*
 
+#### R-NFR-004: Maintenance
+
+- In the long term, the system should be easy to maintain.
+- Not only for the developers but also for the researchers using the system.
+- Origin: Researcher *Goliath*
```

Comment:
maybe one could add a note here that simplicity could lead to better maintainability
e.g. only support one deployment, but support that one well

Reply:
Absolutely, I will add that!